partner_gender/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Gender
+==============
+
+This module was written to extend the functionality of `base_contact`_ to
```

Comment:
Anyways, I see that the field is added to res.partner, so there's no problem to inherit from other view and keep the name, and those that don't like _base_contact_ (like me), can enjoy this module.